src/OrchardCore.Modules/OrchardCore.Metadata/Views/MetadataTextField.Edit.cshtml

```diff
@@ -0,0 +1,17 @@
+@model OrchardCore.Metadata.ViewModels.EditMetadataTextFieldViewModel
+@using OrchardCore.Metadata.Settings
+@using OrchardCore.ContentManagement.Metadata.Models
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<MetadataTextFieldSettings>();
+    var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
+}
+<script asp-src="~/OrchardCore.Metadata/Scripts/character-count.min.js" debug-src="~/OrchardCore.Metadata/Scripts/character-count.js" at="Foot"></script>
```

Comment:
No `~` I think as it would otherwise append the tenant prefix, and break the url. OR did you find another instance of this in another view?